apps/pkeyutl.c

```diff
@@ -322,10 +322,11 @@ int MAIN(int argc, char **argv)
                              buf_in, (size_t)buf_inlen);
         if (rv == 0)
             BIO_puts(out, "Signature Verification Failure\n");
-        else if (rv == 1)
+        else if (rv == 1) {
             BIO_puts(out, "Signature Verified Successfully\n");
-        if (rv >= 0)
-            goto end;
+            ret = 0;
+        }
+        goto end;
```

Comment:
EVP_PKEY_verify() may return a -ve response which previously printed out an error message below so this change is not correct.